etl/steps/data/meadow/gcp/2023-04-28/global_carbon_budget.py

```diff
@@ -0,0 +1,241 @@
+"""Load a snapshot and create a meadow dataset.
+
+It combines the following snapshots:
+- GCP's Fossil CO2 emissions (long-format csv).
+- GCP's official GCB global emissions (excel file) containing global bunker fuel and land-use change emissions.
+- GCP's official GCB national emissions (excel file) containing consumption-based emissions for each country.
+  - Production-based emissions from this file are also used, but just to include total emissions of regions
+    according to GCP (e.g. "Africa (GCP)") and for sanity checks.
+- GCP's official GCB national land-use change emissions (excel file) with land-use change emissions for each country.
+
+"""
+
+import pandas as pd
+from owid.catalog import Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+from etl.snapshot import Snapshot
+
+# Initialize logger.
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def prepare_fossil_co2(df_fossil_co2: pd.DataFrame) -> Table:
+    # Set an appropriate index and sort conveniently.
+    df_fossil_co2 = df_fossil_co2.set_index(["Country", "Year"], verify_integrity=True).sort_index().sort_index(axis=1)
+
+    # Create a new table and ensure all columns are snake-case.
+    tb_fossil_co2 = Table(df_fossil_co2, short_name="global_carbon_budget_fossil_co2_emissions", underscore=True)
+
+    return tb_fossil_co2
+
+
+def prepare_historical_budget(df_historical_budget: pd.DataFrame) -> Table:
```

Comment:
In this case, I'd lean towards function names like `make_table_historical_budget` (also in the following functions).

Reply:
The function's output will always be a `Table`, right? If it was not, I guess it would be a different function. I think it is slightly better than having generic names. But no big deal.